src/App.svelte

```diff
@@ -45,17 +47,45 @@
 	});
 
 	const keepMaganeInPlace = () => {
-		const el = document.querySelector(elementToCheck);
-		if (!el) {
-			if (showIcon) showIcon = false;
-			return;
+		setTimeout(() => {
+			const el = document.querySelector(elementToCheck);
+			if (!el) {
+				if (showIcon) showIcon = false;
+				return;
+			}
+			if (!showIcon) showIcon = true;
+			const props = el.getBoundingClientRect();
+			coords.top = isThereTopBar ? props.top - 21 : props.top;
+			coords.left = props.left - 107;
+		}, 0);
+	};
+
+	const waitForTextArea = () =>
+		new Promise(resolve => {
+			(function pollForTextArea() {
+				textArea = document.querySelector(selectorTextArea);
+				if (textArea) return resolve();
+				setTimeout(pollForTextArea, 500);
+			})();
+		});
+		
+	const positionMagane = entries => {
+		for (const entry of entries) {
+			if (entry.contentRect) {
+				keepMaganeInPlace();
+				if (!entry.contentRect.width && !entry.contentRect.height) {
+					resizeObserver.unobserve(textArea)
```

Comment:
Missing semi-colon and add `// eslint-disable-line no-use-before-define` at the end of the line